### PR TITLE
GNSS RGHL receiver swap

### PR DESCRIPTION
**Pull request by megankortink**
Swap NetR9 for an Alloy

### CODE REVIEW

File: install/receivers.csv [hunk 1]
```diff
@@ -151,6 +151,7 @@ Trimble Navigation Ltd.,TRIMBLE ALLOY,5946R40033,KAPT,2020-02-11T20:15:38Z,9999-
 Trimble Navigation Ltd.,TRIMBLE ALLOY,5946R40034,TGTK,2020-01-29T21:48:38Z,9999-01-01T00:00:00Z
 Trimble Navigation Ltd.,TRIMBLE ALLOY,5946R40069,SCTY,2020-06-23T01:06:32Z,2020-10-27T23:29:07Z
 Trimble Navigation Ltd.,TRIMBLE ALLOY,5946R40072,MKNO,2020-02-12T03:08:38Z,9999-01-01T00:00:00Z
+Trimble Navigation Ltd.,TRIMBLE ALLOY,5951R40056,RGHL,2021-06-01T21:46:08Z,9999-01-01T00:00:00Z
```
Comment (elidana) [inline install/receivers.csv:154]: can you please update the sn to `5946R40026`?

File: install/firmware.csv [hunk 2]
```diff
@@ -496,6 +496,7 @@ Trimble Navigation Ltd.,TRIMBLE ALLOY,5946R40034,5.45,2020-09-28T08:36:22Z,9999-
 Trimble Navigation Ltd.,TRIMBLE ALLOY,5946R40069,5.45,2020-06-23T01:06:32Z,9999-01-01T00:00:00Z,
 Trimble Navigation Ltd.,TRIMBLE ALLOY,5946R40072,5.43,2020-02-12T03:08:38Z,2020-09-28T04:50:04Z,
 Trimble Navigation Ltd.,TRIMBLE ALLOY,5946R40072,5.45,2020-09-28T04:50:05Z,9999-01-01T00:00:00Z,
+Trimble Navigation Ltd.,TRIMBLE ALLOY,5951R40056,5.45,2021-06-01T21:46:08Z,9999-01-01T00:00:00Z,
```
Comment (elidana) [inline install/firmware.csv:499]: can you please update the sn to `15946R40026`?

**Review by elidana — CHANGES_REQUESTED** [review]
Hi Megan,
all good but I've noticed that the receiver serial number has a typo, can you please fix this? Thanks :)

**Comment by elidana** [comment]
and I think that the new receiver `5946R40026` also need to be included in the assets/receivers.csv file

File: install/firmware.csv (hunk 2 above)
Comment (megankortink) [reply install/firmware.csv]: Done

File: install/receivers.csv (hunk 1 above)
Comment (megankortink) [reply install/receivers.csv]: Done

**Comment by megankortink** [comment]
@CBurton90 could you please provide me with the asset number for this serial number (5946R40026)? Then I can add it to the assets/receiver file

**Comment by CBurton90** [comment]
@megankortink I cant find an asset number in delta/assets or sit/inventory. I'd say doesn't have one. Just put the entry in to delta/assets and leave the asset number blank it should still pass.

**Review by elidana — APPROVED** [review]
looks good now, thanks!

**Review by CBurton90 — APPROVED** [review]

**Pull request merged**